boards/grand_central_m4/Cargo.toml

```diff
@@ -57,6 +57,16 @@ chip = "ATSAMD51P20A"
 [[example]]
 name = "blinky_basic"
 
+[[example]]
+name = "neopixel_blink"
+
+[[example]]
+name = "neopixel_rainbow"
+
 [[example]]
 name = "usb_serial"
 required-features = ["usb"]
+
+# remove once the hal is updated
+[patch.crates-io]
+atsamd-hal = { path = "../../hal" }
```

Comment:
Are you going to break out the Hal changes into a separate PR and get it in before v0.15? cc. #570

Reply:
I'd be happy to if you think it is a good idea.

Reply:
Yeah, I think it's the only way to get these changes in for this tier two BSP

Reply:
Hal changes in https://github.com/atsamd-rs/atsamd/pull/588